pkg/application/application.go

```diff
@@ -207,8 +206,8 @@ func SetCurrent(client *occlient.Client, name string) error {
 	return nil
 }
 
-func Exists(client *occlient.Client, name string) (bool, error) {
-	apps, err := List(client)
+func Exists(client *occlient.Client, project, name string) (bool, error) {
```

Comment:
Every function should follow the same order of the arguments: component name, application name, project name.